### ISSUE TITLE
Gomuks isn't receiving new keys?

### ISSUE THREAD

**Issue by ghost** [tulir/gomuks#380]
I'm moving from element to gomuks, and importing the keys from element has gone nicely, and the messages are all viewable. That is, until some time after importing keys, new messages from others fail with "failed to decrypt megolm event: no session with given ID found (ID [id])"
I have a verified session. I can decrypt the messages if I routinely export keys from element and import them to gomuks, but this would be incredibly obnoxious to keep up. Is there something that I need to do to not have to reimport keys?

**Comment by jficz** [comment]
I've started to get random encryption-key related messages about two days ago. The mentioned `megolm event` error above, and also errors from one of my bridges: `Your message was not bridged: failed to get group session: group session has been withheld: m.unauthorized (This device does not share keys to other users)`. 

I don't have problems with any other client I use. The only thing that helps is "factory reset" of Gomuks, setting up everything from scratch with new device ID and reimporting the keys. Tried deleting cryptdb and clear cache but that doesn't help.
